src/test/lambda/explorer/cloudFormationNodes.test.ts

```diff
@@ -194,17 +184,72 @@ describe('DefaultCloudFormationStackNode', () => {
         assert.strictEqual((childNodes[1] as DefaultCloudFormationFunctionNode).label, lambda3Name)
     })
 
+    function validateIconPath(
+        node: TreeItem
+    ) {
+        const fileScheme: string = 'file'
+        const expectedPrefix = `/${fakeIconPathPrefix}/`
+
+        assert(node.iconPath !== undefined)
+        const iconPath = node.iconPath! as {
+            light: Uri,
+            dark: Uri
+        }
+
+        assert(iconPath.light !== undefined)
+        assert(iconPath.light instanceof Uri)
```

Comment:
Out of curiosity, what's the point of the `instanceof` check?

Reply:
I pulled this back from #220 

`iconPath` is defined as `iconPath?: string | Uri | { light: string | Uri; dark: string | Uri } | ThemeIcon;` so I believe this rules out the other types the property could be